client/src/components/Navbar.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { NavLink, Link } from "react-router-dom";
+import styled from "styled-components";
+
+const NavBox = styled.div`
+  width: 100vw;
+  position: fixed;
+  background: white;
+  z-index: 1000;
+`;
+
+const Nav = styled.nav`
+  display: flex;
+  padding: 2rem;
+  box-shadow: 0 -2px 8px rgba(0, 0, 0, 0.6);
+`;
+
+const StyledLink = styled(Link)`
+  font-weight: bolder;
+  text-decoration: none;
+  color: black;
+  cursor: pointer;
+
+  &:hover {
+    opacity: 0.8;
+  }
+`;
+
+const StyledNavLink = styled(NavLink)`
+  margin-left: auto;
+  text-decoration: none;
+  color: black;
+  cursor: pointer;
+
+  &:hover {
+    opacity: 0.8;
+  }
+`;
+
+const Span = styled.span`
+  font-weight: lighter;
+  opacity: 0.8;
+`;
+
+function Navbar() {
```

Comment:
NavBar is very slick. I know they say developers with a true eye for design are considered unicorns, you are a unicorn my friend.